test/telemetry/build.gradle

```diff
@@ -0,0 +1,21 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ *
+ * Modifications Copyright OpenSearch Contributors. See
+ * GitHub history for details.
+ */
+
+apply plugin: 'opensearch.build'
+apply plugin: 'opensearch.publish'
+
+dependencies {
+  api project(":libs:opensearch-common")
+  api project(":libs:opensearch-telemetry")
+  api project(":server")
```

Comment:
So the plugin needs `:server` however the only consumer of the plugin is test framework - we should move it there I believe.

Reply:
Moved the plugin to :test:framework.